tests/framework/utils.py

```diff
@@ -52,7 +53,7 @@ def set_cpu_affinity(pid: int, cpulist: list) -> list:
         return psutil.Process(pid).cpu_affinity(real_cpulist)
 
     @staticmethod
-    def get_cpu_percent(pid: int) -> float:
+    def get_cpu_percent(pid: int) -> Dict[str, Dict[str, float]]:
```

Comment:
https://github.com/firecracker-microvm/firecracker/pull/2227/commits/d6961e77af73471b9cc189f06f1354dbc3eabd5f this seems to be the initial commit for this function which means the return type has been incorrect since the beginning. Since python3 doesn't check the return type annotations (AFAIK) should we have some static analyzer to find more of these incorrect return types?

Reply:
oh whoops, forgot to respond here earlier. 

I don't think there's something that works 100% for python, given its dynamic typing. I supposed there might be a lint along the lines "this function is annotated as returning `float`, but you're treating the return value as a dict`, but I'm not sure if even that is possible without abundant false-positives

Reply:
There is a standard static type checker, mypy, but I think if we add it now we will get tons of errors.